samples/js/text_generation/package-lock.json

```
@@ -9,31 +9,7 @@
       "version": "1.0.0",
       "license": "Apache-2.0",
       "devDependencies": {
-        "openvino-genai-node": "file:../../../src/js/",
-        "openvino-node": "^2025.0.0"
-      },
-      "engines": {
-        "node": ">=21.0.0"
-      }
-    },
-    "../../../src/js": {
-      "name": "openvino-genai-node",
-      "version": "2025.1.0-preview",
-      "dev": true,
-      "hasInstallScript": true,
-      "license": "Apache-2.0",
-      "os": [
-        "linux",
-        "darwin",
-        "win32"
-      ],
-      "dependencies": {
-        "openvino-node": "^2025.0.0"
-      },
-      "devDependencies": {
-        "@huggingface/hub": "^0.21.0",
-        "global-agent": "^3.0.0",
-        "node-fetch": "^3.3.2"
+        "openvino-genai-node": "2025.1.0-preview"
```

```suggestion
        "openvino-genai-node": "2025.2.0"
```
Similar to src/js/package-lock.json

Same belo